cluster/common.sh

```diff
@@ -446,8 +446,19 @@ function yaml-quote {
 # Builds the RUNTIME_CONFIG var from other feature enable options (such as
 # features in alpha)
 function build-runtime-config() {
-  # There is nothing to do here for now. Just using this function as a placeholder.
-  :
+  # If a policy provider is specified, enable NetworkPolicy API.
+  if [[ -n "${NETWORK_POLICY_PROVIDER}" ]]; then
```

Comment:
OK - didn't get a chance to try AWS myself, but here is a fix: https://github.com/kubernetes/kubernetes/pull/28256